packages/components/src/components/hds/copy/button/types.ts

```diff
@@ -0,0 +1,13 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+export enum HdsCopyButtonSizeValues {
+  Small = 'small',
+  Medium = 'medium',
+}
+export type HdsCopyButtonSizes = `${HdsCopyButtonSizeValues}`;
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export type HdsCopyButtonItemToCopy = string | ((...args: any[]) => void);
```

Comment:
As mentioned above, I don't think you need this, the type(s) should come from the signature of the `hds-clipboard` modifier

Reply:
Removed